libs/sdk-core/src/greenlight.rs

```diff
@@ -537,17 +537,20 @@ async fn pull_transactions(
         .await?
         .into_inner();
     debug!("list payments: {:?}", payments);
-    // construct the payment transactions
-    let sent_transactions: Result<Vec<crate::models::Payment>> = payments
+    // construct the payment transactions (pending and complete)
+    let outbound_transactions: Result<Vec<crate::models::Payment>> = payments
         .payments
         .into_iter()
-        .filter(|p| p.created_at as i64 > since_timestamp && p.status() == PayStatus::Complete)
+        .filter(|p| {
```

Comment:
We now add to the db also pending payments and this means that any code that was using the `get_payment_by_hash` function is now have different meaning because now also pending payments may return.
Specificaly the regular swap use that function so I think we should check for the right state there.

Reply:
Is this an acceptable solution? 87c0f1624ecfd8142c4d13ebee6bb84c2b59d290

Reply:
Yes.